pkg/entrypoint/entrypointer.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2019 The Tekton Authors
+Copyright 2019, 2020 The Tekton Authors
```

Comment:
I don't think we need to worry about this, at least not in this change. If we decide we care we can make one automated change across the whole repo.

Reply:
done.